neo/Network/P2P/Payloads/Cosigner.cs

```diff
@@ -32,6 +32,8 @@ void ISerializable.Deserialize(BinaryReader reader)
         {
             Account = reader.ReadSerializable<UInt160>();
             Scopes = (WitnessScope)reader.ReadByte();
+            if (!Enum.IsDefined(typeof(WitnessScope), Scopes))
```

Comment:
`WitnessScope` has a `Flags` attribute, so it can't be check in this way.

Reply:
it was a suggestion by shargon, but i'll remove it again.